### PR TITLE
Update mondo.md description

### LINKED ISSUE #2629: MONDO download files no longer working
There appears to be a problem with the generated MONDO download files for the past few days.  The path:

http://purl.obolibrary.org/obo/mondo/mondo-with-equivalents.json

No longer resolves.  Nor does the link on the ontology page.  https://obofoundry.org/ontology/mondo.html

I’m contacting you since you are listed as the contact on the page.  If there is a different contact, please let me know.

Thanks!

Regards,

Phil Weller

### PR DESCRIPTION
**Pull request by matentzn**
Fixes #2629

Needs @sabrinatoro @twhetzel review and editorial intervention.

### CODE REVIEW

**Review by sabrinatoro — APPROVED** [review]
Thank you @matentzn. This looks good to me!

File: ontology/mondo.md [hunk 1]
```diff
@@ -87,12 +87,11 @@ usages:
 activity_status: active
 ---
 
-The Mondo Disease Ontology (Mondo) aims to harmonize disease definitions across the world. It is a global community effort to reconcile and curate the very many disease resources within a coherent merged ontology. Original versions of Mondo were constructed entirely automatically and used the IDs of source databases and ontologies. Later, additional manually curated cross-ontology axioms were added, and a native Mondo ID system was used to avoid confusion with source databases.
+The Mondo Disease Ontology (Mondo) aims to harmonize existing disease terminologies into a coherent ontological representation of diseases.
+Mondo integrates widely used terminologies such as Online Mendelian Inheritance in Man (OMIM), Orphanet, Experimental Factor Ontology (EFO), Disease Ontology (DOID) and the neoplasm branch of National Cancer Institute Thesaurus (NCIt).
```
Comment (twhetzel) [inline ontology/mondo.md:91]: Should ICD11 be mentioned?

**Review by twhetzel — APPROVED** [review]
Added one question. Otherwise looks fine.

File: ontology/mondo.md (hunk 1 above)
Comment (matentzn) [reply ontology/mondo.md:91]:
```suggestion
Mondo integrates widely used terminologies such as Online Mendelian Inheritance in Man (OMIM), Orphanet, Experimental Factor Ontology (EFO), Disease Ontology (DOID), ICD 11 (Foundation) and the neoplasm branch of National Cancer Institute Thesaurus (NCIt).
```

**Review by deepakunni3 — APPROVED** [review]

**Pull request merged**